### PR TITLE
Added metrics collection and metrics-based dashboards info to MongoDB

### PR DESCRIPTION
**Pull request by yasar-sumologic**
## Purpose of this pull request

This pull request Adds metrics collection and metrics-based dashboards info to MongoDB 

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## Ticket (if applicable)

https://sumologic.atlassian.net/browse/SUMO-243921

### CODE REVIEW

File: docs/integrations/databases/opentelemetry/mongodb-opentelemetry.md
Comment (sumoanema) [inline docs/integrations/databases/opentelemetry/mongodb-opentelemetry.md:14]: We can specify the version of mongo which we have tested against.

File: docs/integrations/databases/opentelemetry/mongodb-opentelemetry.md
Comment (sumoanema) [inline docs/integrations/databases/opentelemetry/mongodb-opentelemetry.md:42]: Will need to add instruction for profiling level setting in mongo for sending logs.

File: docs/integrations/databases/opentelemetry/mongodb-opentelemetry.md
Comment (sumoanema) [inline docs/integrations/databases/opentelemetry/mongodb-opentelemetry.md:99]: We should mention that username/password is optional

File: docs/integrations/databases/opentelemetry/mongodb-opentelemetry.md
Comment (sumoanema) [inline docs/integrations/databases/opentelemetry/mongodb-opentelemetry.md:102]: We should include this in logs PAth description itself and not mention this line seperately.

File: docs/integrations/databases/opentelemetry/mongodb-opentelemetry.md
Comment (sumoanema) [inline docs/integrations/databases/opentelemetry/mongodb-opentelemetry.md:280]: Do we have a sharding dashboard ?

File: docs/integrations/databases/opentelemetry/mongodb-opentelemetry.md
Comment (sumoanema) [inline docs/integrations/databases/opentelemetry/mongodb-opentelemetry.md:293]: Have we made changes to this dashboard . This seem to be a log dashbaord.. and we have added replication related metric dashboard seperately. Why are we changing the description for this dashboard ?

File: docs/integrations/databases/opentelemetry/mongodb-opentelemetry.md
Comment (yasar-sumologic) [reply docs/integrations/databases/opentelemetry/mongodb-opentelemetry.md:42]: it is already there on line number 46

File: docs/integrations/databases/opentelemetry/mongodb-opentelemetry.md
Comment (yasar-sumologic) [reply docs/integrations/databases/opentelemetry/mongodb-opentelemetry.md:102]: fixed it.

File: docs/integrations/databases/opentelemetry/mongodb-opentelemetry.md
Comment (yasar-sumologic) [reply docs/integrations/databases/opentelemetry/mongodb-opentelemetry.md:280]: Yes it is already there & it is logged based dashboard

File: docs/integrations/databases/opentelemetry/mongodb-opentelemetry.md
Comment (yasar-sumologic) [reply docs/integrations/databases/opentelemetry/mongodb-opentelemetry.md:293]: fixed it.

File: docs/integrations/databases/opentelemetry/mongodb-opentelemetry.md
Comment (yasar-sumologic) [reply docs/integrations/databases/opentelemetry/mongodb-opentelemetry.md]: fixed it

File: docs/integrations/databases/opentelemetry/mongodb-opentelemetry.md
Comment (yasar-sumologic) [reply docs/integrations/databases/opentelemetry/mongodb-opentelemetry.md]: fixed it.

File: docs/integrations/databases/opentelemetry/mongodb-opentelemetry.md
Comment (sumoanema) [reply docs/integrations/databases/opentelemetry/mongodb-opentelemetry.md:42]: That is log level. We need to add instruction for setting profiling level

File: docs/integrations/databases/opentelemetry/mongodb-opentelemetry.md
Comment (sumoanema) [inline docs/integrations/databases/opentelemetry/mongodb-opentelemetry.md:106]: We also need to mention if authentication for mongo is configured then they need to download env file. Please check the content in any of the other app docs.

File: docs/integrations/databases/opentelemetry/mongodb-opentelemetry.md
Comment (sumoanema) [inline docs/integrations/databases/opentelemetry/mongodb-opentelemetry.md:101]: For username and password, can we mention something on the lines of mongodb receiver doc : 
**username (optional)** : If authentication is required, the user can with **clusterMonitor** permissions can be provided here.
**password (optional)**: If authentication is required, the password can be provided here.

File: docs/integrations/databases/opentelemetry/mongodb-opentelemetry.md
Comment (sumoanema) [reply docs/integrations/databases/opentelemetry/mongodb-opentelemetry.md:282]: So was the doc missing the description for sharding dashboard ?

File: docs/integrations/databases/opentelemetry/mongodb-opentelemetry.md
Comment (sumoanema) [reply docs/integrations/databases/opentelemetry/mongodb-opentelemetry.md:104]: I do not see it fixed. You need to remove this line from here and place it under "Logs Path"

File: docs/integrations/databases/opentelemetry/mongodb-opentelemetry.md
Comment (yasar-sumologic) [reply docs/integrations/databases/opentelemetry/mongodb-opentelemetry.md:106]: fixed it.

File: docs/integrations/databases/opentelemetry/mongodb-opentelemetry.md
Comment (yasar-sumologic) [reply docs/integrations/databases/opentelemetry/mongodb-opentelemetry.md]: fixed it.

File: docs/integrations/databases/opentelemetry/mongodb-opentelemetry.md
Comment (yasar-sumologic) [reply docs/integrations/databases/opentelemetry/mongodb-opentelemetry.md:104]: fixed it.

Comment (sumoanema) [reply docs/integrations/databases/opentelemetry/mongodb-opentelemetry.md:104]: Can you remove line 104. There is no use of it after we have added details above

File: docs/integrations/databases/opentelemetry/mongodb-opentelemetry.md
Comment (sumoanema) [reply docs/integrations/databases/opentelemetry/mongodb-opentelemetry.md:106]: in step3 after this there should be steps to include env file under linux tab. Please refer mysql doc for it.

File: docs/integrations/databases/opentelemetry/mongodb-opentelemetry.md
Comment (yasar-sumologic) [reply docs/integrations/databases/opentelemetry/mongodb-opentelemetry.md:104]: Done

File: docs/integrations/databases/opentelemetry/mongodb-opentelemetry.md
Comment (yasar-sumologic) [reply docs/integrations/databases/opentelemetry/mongodb-opentelemetry.md:106]: Ok

File: docs/integrations/databases/opentelemetry/mongodb-opentelemetry.md
Comment (JV0812) [inline docs/integrations/databases/opentelemetry/mongodb-opentelemetry.md:14]:
```suggestion
[MongoDB](https://www.mongodb.com/why-use-mongodb#:~:text=MongoDB%20is%20a%20document%20database,development%20teams%20using%20agile%20methodologies.) is a source-available cross-platform document-oriented database program. The Sumo Logic app for MongoDB supports logs and metrics from the open source version of MongoDB. The app is tested on the 7.0.14 version of MongoDB.
```

File: docs/integrations/databases/opentelemetry/mongodb-opentelemetry.md
Comment (JV0812) [inline docs/integrations/databases/opentelemetry/mongodb-opentelemetry.md:22]:
```suggestion
This app supports logs and metrics for MongoDB instance. The MongoDB logs are generated in files as configured in this configuration file `/var/log/mongodb/mongodb.log`. For more details on MongoDB logs, refer to the [MongoDB documentation](https://docs.mongodb.com/manual/reference/log-messages/).
```

File: docs/integrations/databases/opentelemetry/mongodb-opentelemetry.md
Comment (JV0812) [inline docs/integrations/databases/opentelemetry/mongodb-opentelemetry.md:49]:
```suggestion
2. You need to set the [profiling_level](https://www.mongodb.com/docs/manual/reference/method/db.setProfilingLevel/) to a value of 1. But by default it is set to 0. You can also configure the value of [slowms](https://www.mongodb.com/docs/manual/reference/method/db.setProfilingLevel/#std-label-set-profiling-level-options-slowms) to a value greater than or equal to 100. Thus can be done using the below command in mongosh:
```

File: docs/integrations/databases/opentelemetry/mongodb-opentelemetry.md
Comment (yasar-sumologic) [reply docs/integrations/databases/opentelemetry/mongodb-opentelemetry.md]: fixed it.

File: docs/integrations/databases/opentelemetry/redis-opentelemetry.md
Comment (JV0812) [inline docs/integrations/databases/opentelemetry/redis-opentelemetry.md:121]:
```suggestion
2. Move the env file in the following directory:
```

File: docs/integrations/databases/opentelemetry/mongodb-opentelemetry.md
Comment (JV0812) [inline docs/integrations/databases/opentelemetry/mongodb-opentelemetry.md:294]:
```suggestion
- Identify sharding errors and warnings.
```

**Review by JV0812 — APPROVED** [review]
LGTM

**Review by jpipkin1 — APPROVED** [review]
Approved, @yasar-sumologic. I made minor changes (see commit "[Updates for review](https://github.com/SumoLogic/sumologic-documentation/pull/4517/commits/0be17a7c2ce196db628e93cbf5781cc5cf95a599)").

**Comment by jpipkin1** [comment]
@yasar-sumologic, is this PR ready to be published? 

If instead it needs to wait until a software release, add the **do-not-merge** label.

**Comment by jpipkin1** [comment]
@yasar-sumologic confirmed via Slack that this PR is ready to be published.

**Pull request merged**
